src/style.css

```diff
@@ -149,6 +149,19 @@ li {
   margin-top: -5px;
 }
 
+/*
+.pos-rel {
+  position: relative;
+}
+
+.desc {
+  width: 100%;
+  border: 0;
+  outline: 0;
+  background-color: transparent;
+}
+*/
+
```

Comment:
- [ ] Kindly remove all the commented out code for efficiency in your codebase and for easy understanding.